src/common_components/header/HeaderButton.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+import { css } from "@styled-system/css";
+import Image from "next/image";
+import { FC, MouseEventHandler, ReactNode } from "react";
+
+export const HeaderButton: FC<{
+  text: ReactNode;
+  icon: string;
+  clickev: MouseEventHandler<HTMLButtonElement>;
+}> = ({ text, icon, clickev }) => (
+  <button
+    className={css({
+      cursor: "pointer",
+      fontSize: "sm",
+      px: { lg: 5, base: 0 },
+      width: "fit-content",
+      height: "100%",
```

Comment:
```suggestion
      height: "full",
```
Panda CSSの[theme](https://panda-css.com/docs/customization/theme)を使うように統一しているのでお願いします